app/routers/dayview.py

```diff
@@ -0,0 +1,112 @@
+    BASE_GRID_BAR = 5
+    FIRST_GRID_BAR = 1
+    LAST_GRID_BAR = 101
+
+    def __init__(self, event: Event,
+                 day: Union[bool, datetime] = False) -> None:
+        self.start_time = event.start
+        self.end_time = event.end
+        self.day = day
+        self.start_multiday, self.end_multiday = self._check_multiday_event()
+        self.color = self._check_color(event.color)
+        self.total_time = self._set_total_time()
+        self.grid_position = self._set_grid_position()
+
+    def _check_color(self, color: str) -> str:
+        if color is None:
+            return 'grey'
+        return color
+
+    def _minutes_position(self, minutes: int) -> int:
+        for i in range(self.GRID_BAR_QUARTER, self.FULL_GRID_BAR + 1):
+            if self.MIN_MINUTES < minutes <= self.MAX_MINUTES:
+                return i
+            self.MIN_MINUTES = self.MAX_MINUTES
+            self.MAX_MINUTES += 15
+
+    def _get_position(self, time: datetime) -> int:
+        grid_hour_position = time.hour * self.FULL_GRID_BAR
+        grid_minutes_modifier = self._minutes_position(time.minute)
+        if grid_minutes_modifier is None:
+            grid_minutes_modifier = 0
+        return grid_hour_position + grid_minutes_modifier + self.BASE_GRID_BAR
+
+    def _set_grid_position(self) -> str:
+        if self.start_multiday:
+            start = self.FIRST_GRID_BAR
+        else:
+            start = self._get_position(self.start_time)
+        if self.end_multiday:
+            end = self.LAST_GRID_BAR
+        else:
+            end = self._get_position(self.end_time)
+        return f'{start} / {end}'
+
+    def _get_time_format(self) -> str:
+        format = "%H:%M"
+        multiday_format = "%d/%m %H:%M"
+        for multiday in [self.start_multiday, self.end_multiday]:
+            if multiday:
+                yield multiday_format
+            else:
+                yield format
+
+    def _set_total_time(self) -> None:
+        length = self.end_time - self.start_time
+        self.length = length.seconds / 60
+        format_gen = self._get_time_format()
+        start_time_str = self.start_time.strftime(next(format_gen))
+        end_time_str = self.end_time.strftime(next(format_gen))
+        return ' '.join([start_time_str, '-', end_time_str])
+
+    def _check_multiday_event(self) -> Tuple[bool]:
+        start_multiday, end_multiday = False, False
+        if self.day:
+            if self.start_time < self.day:
+                start_multiday = True
+            self.day += timedelta(hours=24)
+            if self.day <= self.end_time:
+                end_multiday = True
+        return (start_multiday, end_multiday)
+
+
+@router.get('/day/{date}')
+async def dayview(request: Request, date: str, db_session=Depends(get_db)):
+    # temporary fake user until there will be login session
+    user = db_session.query(User).filter_by(username='test1').first()
+    day = datetime.strptime(date, '%d-%m-%Y')
```

Comment:
I would like to assume that we will use %Y-%m-%d format because it's the only correct format (and international standard thing: https://en.wikipedia.org/wiki/ISO_8601)